code/modules/client/client_procs.dm

```diff
@@ -195,11 +195,11 @@ var/next_external_rsc = 0
 	sethotkeys(1)						//set hoykeys from preferences (from_pref = 1)
 
 	. = ..()	//calls mob.Login()
-
+	
```

Comment:
Ya, i meant to copy paste the line to here, but when i tabbed back i put it elsewhere.

It happens,